api/src/main/java/org/hawkular/apm/api/model/trace/CorrelationIdentifier.java

```diff
@@ -129,10 +129,27 @@ public String toString() {
     public enum Scope {
 
         /**
-         * The 'Interaction' scope means the identifier is only unique in relation
+         * This scope means the identifier is only unique in relation
          * to a specific exchange between two (or more) participants.
          */
-        Interaction
+        Interaction,
+
+        /**
+         * This scope means the identifier is a unique value shared between
+         * one source node and potentially zero or more target nodes. The
+         * value can be randomly generated (as in the case of establishing
+         * a link between concurrent fragments) or represent some application/
+         * business context (e.g. order id). This value must be unique within
```

Comment:
`order id`? Is it safe to say that id based on business data is globally unique?